### DISCUSSION TITLE
Field Value in Return URL

### DISCUSSION THREAD

**Discussion by kevinamezaga** [solspace/craft-freeform#195] [Questions & Answers] [answered]
So I want to send a user back to a specific entry related to their form submission after they submit the form. What's the best way to do this using the return URL? I noticed you can't use fields like you can in the submission title.

**Comment by kjmartens** [comment, marked as answer]
Hi @kevinamezaga,

If I'm understanding your request correctly, you'd like to be able to have the user taken to different URL's/pages (perhaps Craft entries) based on the selection of an option in a certain field?

If so, the closest you might be able to come to that would be using this feature:
https://docs.solspace.com/craft/freeform/v3/guides/success-return-on-forms.html#user-chooses-return-url

Or, if you're wanting to display some or all of their submission data on the success page, check out this guide here:
https://docs.solspace.com/craft/freeform/v3/guides/submission-data-success.html
